### PR TITLE
Nikita Grechuhin lab1

### PR DESCRIPTION
**Pull request by nik709**

### CODE REVIEW

**Comment by nik709** [comment]
@PovelikinRostislav
@Vagik
@komarovn

File: modules/complex-number/test/test_grechuhin_nikta_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    ComplexNumber a(re1, im1);
+    ComplexNumber b(re2, im2);
+    ComplexNumber c;
+    c = a*b;
+
+    EXPECT_EQ(re1*re2, c.getRe());
+    EXPECT_EQ(re1*im2, c.getIm());
+}
+
+TEST(Grechuhin_Nikita_Complex_Number_Test, Can_summerize_numbers) {
```
Comment (PovelikinRostislav) [inline modules/complex-number/test/test_grechuhin_nikta_complex_number.cpp]: We can't make a summEr via complex numbers.

File: modules/complex-number/test/test_grechuhin_nikta_complex_number.cpp [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    double re1 = 2.0;
+    double im1 = 0.0;
+    double re2 = 1.5;
+    double im2 = 5.6;
+
+    // Act
+    ComplexNumber a(re1, im1);
+    ComplexNumber b(re2, im2);
+    ComplexNumber c;
+    c = a*b;
```
Comment (PovelikinRostislav) [inline modules/complex-number/test/test_grechuhin_nikta_complex_number.cpp]: Please add spaces between operators to make code more in a style.

File: modules/complex-number/test/test_grechuhin_nikta_complex_number.cpp [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+    // Act
+    ComplexNumber z(re, im);
+    ComplexNumber test_number(z);
+
+    // Assert
+    EXPECT_EQ(test_number, z);
+}
+
+TEST(Grechuhin_Nikita_Complex_Number_Test, Can_Set_Re_and_Im) {
```
Comment (PovelikinRostislav) [inline modules/complex-number/test/test_grechuhin_nikta_complex_number.cpp]: What is your style? 
Some words are started from upper-case, some not
Choose the one way please!

File: modules/complex-number/test/test_grechuhin_nikta_complex_number.cpp [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    EXPECT_EQ(im, z.getIm());
+}
+
+TEST(Grechuhin_Nikita_Complex_Number_Test, Equals_operator_is_correct) {
+    // Arrange
+    double re = 1.2;
+    double im = 2.1;
+
+    // Act
+    ComplexNumber z1(re, im);
```
Comment (PovelikinRostislav) [inline modules/complex-number/test/test_grechuhin_nikta_complex_number.cpp]: The same is with variables. 
In prev test you use 'test_number'
Here you use 'z1,z2'
In next - 'a,b,c'

Please use the same names for the same purposes - testing

File: modules/complex-number/test/test_grechuhin_nikta_complex_number.cpp (hunk 1 above)
Comment (PovelikinRostislav) [inline modules/complex-number/test/test_grechuhin_nikta_complex_number.cpp]: >Can_summerize_numbers

**Comment by nik709** [comment]
closed for fix style

**Pull request closed without merging**